showhide.js

```diff
@@ -3,7 +3,7 @@
     var elems = $('*[showhide], *[onclick*=wp_showhide]'),
       $elem, id, text, settings, visible;
 
-    for (var i = 0; i < elems.length; i++) {
+    for (var j = 0; j < elems.length; i++) {

```

Comment:
I tested something different on another site and it worked. I'm still learning Github. Please ignore for now.